workflow/workflow.go

```diff
@@ -0,0 +1,178 @@
+package workflow
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"path/filepath"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/jbsmith7741/go-tools/appenderr"
+	"github.com/pcelvng/task"
+	"github.com/pkg/errors"
+
+	"github.com/pcelvng/task-tools/file"
+)
+
+type Workflow struct {
+	Task      string // doubles as the Name of the topic to send data to
+	Rule      string
+	DependsOn string // Task that the previous workflow depends on
+	Retries   int
+	Template  string // template used to create the task
+}
+
+type loader struct {
```

Comment:
This should be renamed. This is wrapper struct that is used to unmarshal the workflow file. It could be called file, but I think that would cause some confusion. Naming ideas:
- record (workflow.record)
- archive (workflow.archive)

Reply:
maybe workflow job, step, or chore?  Isn't this a workflow task?